proctl/proctl.go

```
@@ -143,13 +192,19 @@ func (dbp *DebuggedProcess) FindLocation(str string) (uint64, error) {
 		return fn.Entry, nil
 	}
 
-	// Attempt to parse as number for breakpoint id or raw address
+	// No joy
+	return 0, nil
```

This should really return an error when no location is actually found.